docs/api.adoc

```diff
@@ -341,6 +341,136 @@ Signs transaction offline with private key from keystore file with given passphr
 <-- {"jsonrpc": "2.0", "result": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675", "id": 1}
 ----
 
+=== emerald_addContract
```

Comment:
May be better to rename it to `emerald_importContract`, to be more consistent with already existing `emerald_importAccount & emerald_exportAccount`, some more reason to do it this way i wrote below, for `emerald_listContracts`.